ui/partials/perf/comparesubtestctrl.html

```diff
@@ -6,10 +6,9 @@
   </div>
   <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
     <h1>{{subtestTitle}} subtest summary</h1>
-    <ul class="list-inline push-information">
-      <li><strong>Base</strong> - <a href="{{originalRevision|getRevisionUrl:originalProject.name}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-      <li><strong>New</strong> - <a href="{{newRevision|getRevisionUrl:newProject.name}}">{{newRevision}}</a> ({{newProject.name}})</li>
-    </ul>
+      <ul class="list-inline push-information">
```

Comment:
As I mentioned earlier, this should be put inside a directive: https://github.com/mozilla/treeherder/pull/795#discussion_r35665365